test/verify/check-services

```diff
@@ -90,7 +98,14 @@ ExecStart=/usr/local/bin/test-service %I
 
         unit_action_btn = '#service-valid .list-group-item:nth-child(1) .btn-group'
         file_action_btn = '#service-valid .list-group-item:nth-child(2) .btn-group'
-
+        # Selects Timer tab
+        b.click('#services-filter :nth-child(4)')
+        b.wait_visible('#services')
+        m.execute("systemctl start test.timer")
+        b.wait_in_text(svc_sel('test.timer'), "Today")
```

Comment:
also, make sure that `svc_sel('test.timer')` is exactly the selector you want to use here